fe/fe-core/src/main/java/org/apache/doris/tablefunction/ExternalFileTableValuedFunction.java

```diff
@@ -227,8 +227,7 @@ protected void parseProperties(Map<String, String> validParams) throws AnalysisE
         }
 
         // TODO Support is needed in the future
-        if (getTFileType() == TFileType.FILE_STREAM && (formatString.equals("json")
-                || formatString.equals("csv_with_names")
+        if (getTFileType() == TFileType.FILE_STREAM && (formatString.equals("csv_with_names")
                 || formatString.equals("csv_with_names_and_types")
                 || formatString.equals("parquet")
                 || formatString.equals("avro")
```

Comment:
throw new AnalysisException("current http_stream does not yet support **json**, parquet and orc");